coreneuron/gpu/nrn_acc_manager.cpp

```diff
@@ -532,66 +532,6 @@ void delete_ivoc_vect_from_device(IvocVect& vec) {
 #endif
 }
 
-void realloc_net_receive_buffer(NrnThread* nt, Memb_list* ml) {
```

Comment:
Not dead, can be called from NMODL/MOD2C:
https://github.com/BlueBrain/nmodl/blob/4a67187535eb6d79d7ef74095dacd6434948a43d/src/codegen/codegen_c_visitor.cpp#L4104